### ISSUE TITLE
Train 'Mission Classifiers'

### ISSUE THREAD

**Issue by Jack-0-0** [nestauk/discovery_utils#14]
Train 3 binary 'Mission Classifiers' i.e one classifier for AHL, AFS, ASF:
- Start with simple keyword matching approach
- Try more advanced methods such as embeddings + classifiers or fine-tuning a transformer model

If more training data is needed, we can use LLMs to label more data.

**Comment by beingkk** [comment]
Thanks for creating the issue @Jack-0-0 !
FYI a simple version of [keyword matching](https://github.com/nestauk/discovery_utils/blob/22bc07312286086b69cd243e798b235a00999576/discovery_utils/enrichment/crunchbase.py#L488) is already implemented in the enrichment pipeline
